package.json

```diff
@@ -628,6 +633,10 @@
         "command": "todo.toggleBox",
         "title": "Todo: Toggle Box"
       },
+      {
+        "command": "todo.boxNextLine",
```

Comment:
This command and "todo.insertNewLine" are pretty useless for the user, I mean I don't see anyone triggering them from the command palette, do we absolutely need them at all? If so, can you hide them from the command palette?